server/service/service_appconfig.go

```diff
@@ -82,48 +81,6 @@ func (svc *Service) sendTestEmail(ctx context.Context, config *fleet.AppConfig)
 	return nil
 }
 
-func (svc *Service) makeTestJiraRequest(ctx context.Context, jiraSettings *fleet.JiraIntegration) error {
```

Comment:
This got moved to the `fleet` package so it can be reused with the `ee` package for team integrations validation.